### PR TITLE
Edits to About, Round 2

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -45,59 +45,58 @@ <h2 class="nodisplay">About</h2>
...
-			<td><h5>2012 - 2014</h5></td>
-			<td>Co-Founder, Product Manager, UX/UI-Designer of <a href="../flying/">Flying - App Startup</a></br><i class="half">Vienna, Hamburg</i></td>
+			<td><h5>2012 – 2014</h5></td>
+			<td>Co-Founder, Product Manager, UX/UI Designer of <a href="../flying/">Flying</a></br><i class="half">Vienna, Hamburg</i></td>
 		</tr>
 		<tr>
-			<td><h5>2012 - 2013</h5></td>
-			<td>studying at <a href="https://ciid.dk/">CIID</a> - Master-level Interaction Design</br><i class="half">Copenhagen</i></td>
+			<td><h5>2012 – 2013</h5></td>
+			<td>Studied Interaction Design at <a href="https://ciid.dk/">CIID — Master</a></br><i class="half">Copenhagen</i></td>
```
Comment (deutschbitte) [inline about/index.html:79]: Did you get a Master of Arts, or? Trying to align this with the other degree

Comment (smeidu) [reply about/index.html:79]: nope, it's a "master level programme" without any official academic title.

File: about/index.html [hunk 2]
```diff
@@ -45,59 +45,58 @@ <h2 class="nodisplay">About</h2>
...
+		<p class="body2">After graduating, I founded <a href="../flying/"another startup</a>, which I brought back to Vienna and developed until 2015. I've always had a professional interest in accessibility, so I also founded an <a href="../blindmaps/">open-source project</a>. Shortly after, Google made me an offer, and I ended up in Munich for four years — with side trips to the Silicon Valley, New York, and Tel Aviv — where I worked with many amazing and smart Google colleagues on <a href="../google/">UI standards, accessibility, and account settings projects across the Google universe</a>. Eventually, I felt compelled to move "home" and returned to Upper Austria and Vienna to found my own <a href="../work/">studio</a><i class="half">... to be continued</i></p>
```
Comment (deutschbitte) [inline about/index.html:56]: Added the link here, though the CV says you did this until 2014, so we should change this or the below to be the same year, depending on which it was

File: about/index.html [hunk 3]
```diff
@@ -45,59 +45,58 @@ <h2 class="nodisplay">About</h2>
...
-			<td><h5>2004 - 2012</h5></td>
-			<td>Webdesigner, Project Manager @ Knallgrau (Web 2.0 Agency)</br><i class="half">Vienna</i></td>
+			<td><h5>2004 – 2012</h5></td>
+			<td>Web Designer, Project Manager @ Knallgrau (Web 2.0 Agency)</br><i class="half">Vienna</i></td>
 		</tr>
 		<tr>
-			<td><h5>2001 - 2004</h5></td>
-			<td>studying Multimedia Arts - Bachelor of Arts</br><i class="half">Vienna</i></td>
+			<td><h5>2001 – 2004</h5></td>
+			<td>Studied Multimedia Arts — Bachelor of Arts</br><i class="half">Vienna</i></td>
```
[the remark this answers is not in the document]
Comment (smeidu) [reply about/index.html:91]: SAE College Vienna in partnership with Middlesex University London.
I am honestly not so sure if I want to state it ;-). Better to leave it out

File: about/index.html (hunk 2 above)
Comment (smeidu) [reply about/index.html:56]: good catch! It should be 2014 everywhere.

**Comment by deutschbitte** [comment]
Great, updated, you can merge and then I'll revisit this later if need be

**Pull request merged**